Remove Upgrade Lab feature flag from usage and config docs (#1219)
chef/chef-workstation: www/content/workstation/config.md

```toml
[features]
example = true
```

Description

> : List of experimental features. Features are not enabled by default. Enable the feature with `name = true` and disable with `name = false`. The above example enables one feature, which is the `example` feature (there are currently no actual features that can be controlled in this manner). You can also enable or disable any feature from the command line using an environment variable. For example, setting `CHEF_FEAT_EXAMPLE=true` from the command line enables the `example` feature for the duration of your terminal session.

Reviewer: ```suggestion
: List of experimental features. Boolean. Default: none. Enable the feature with `feature = true` and disable with `feature = false`.  `example = true` enables one feature, which is the `example` feature. You can also enable or disable any feature from the command line using an environment variable. For example, setting `CHEF_FEAT_EXAMPLE=true` from the command line enables the `example` feature for the duration of your terminal session.
```

Reviewer: Is `CHEF_FEAT_EXAMPLE` passed in the CLI the same value as `example` in the config?

Author: Yes
